samples/list/PageableList-ObservableRest.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta http-equiv="Content-type" content="text/html; charset=utf-8"/>
+	<meta name="viewport"
+		  content="width=device-width,initial-scale=1,maximum-scale=1,minimum-scale=1,user-scalable=no"/>
+	<meta name="apple-mobile-web-app-capable" content="yes"/>
+
+	<title>deliteful/list/List samples.</title>
+
+	<script type="text/javascript" src="../../../requirejs/require.js"></script>
+
+	<script type="text/javascript">
+		require.config({
+			baseUrl: "../../../"
+		});
+	</script>
+
+	<script type="text/javascript">
+		require([
+			"dojo/request",
+			"delite/register",
+			'requirejs-dplugins/Promise!', // page level CSS
+			"deliteful/list/PageableList",
+			"decor/ObservableArray"
+		], function (request, register, Promise, PageableList, ObservableArray) {
+
+			var list  = register.createElement("d-pageable-list");
+			list.id = "L";
+
+			var initRequest = request("http://9.128.82.26:3000/book?&start=" + 0 + "&length=" + 4, {
```

Comment:
Using a variable to store the rest endpoint would allow a user to customize it more easily.